manual/docs/machinery-compare.1.md

```diff
@@ -3,7 +3,7 @@
 
 ## SYNOPSIS
 
-`machinery compare` [-s SCOPE | --scope=SCOPE] [-e EXCLUDE-SCOPE | --exclude-scope=EXCLUDE-SCOPE] [--no-pager] [--show-all] [--html] NAME1 NAME2
+`machinery compare` [-s SCOPE | --scope=SCOPE] [-e IGNORE-SCOPE | --ignore-scope=IGNORE-SCOPE] [--no-pager] [--show-all] [--html] NAME1 NAME2
```

Comment:
Later on I thought so too, but ```-i``` is already reserved for the ```--ssh-identity-file``` option. Maybe we need to discuss this seperately.